src/content/docs/new-relic-solutions/tutorial-landing-page.mdx

```diff
@@ -48,7 +48,7 @@ This page contains all of our tutorials. Whether you're just starting your New R
 
 <DocTiles>
     <DocTile path="/docs/tutorial-reporting-data/capture-the-right-data/" title="Capture the right data">
-        Improve troubleshooting and make informed decisions with the right data.
+        Improve troubleshooting and have informed decisions with the right data.
```

Comment:
@techpubswriter I think the more idiomatic formulation is "make informed" decisions. While we can "have" informed decisions, the context of troubleshooting seems to require a more active verb showing action. Let's stick with the original version with "make."